app/mltraining_client_test.go

```diff
@@ -39,12 +39,7 @@ var (
 		ErrorStatus: &ErrorStatus{
 			Code:    code,
 			Message: message,
-			Details: []*GenericProtoMessage{
-				{
-					TypeURL: siteURL,
-					Value:   byteData,
-				},
-			},
+			Details: []interface{}(nil),
```

Comment:
(q) curious why we move to an empty `Details` here? I think having a specific type in there to reflect what would typically go here would be helpful.